CodeReviews/CodeReviewBadPractices.md

```diff
@@ -1,18 +1,20 @@
 # Bad Practices - Code Reviews
 
 * **Review Code Not People**  
-When reviewing others code it is best to remember that it is not a competition to see who can code better. Software developers are often proud of their solutions and take criticism personally on occasion. Reviews help us see how others thought process occurs and feedback given should be constructive uplifting and insightful to help others program more efficiently. Be careful to always provide feedback that does not question the person but questions how and why they created a solution the way they did.
+When reviewing others code it is best to remember that it is not a competition to see who can code better. Software developers are often proud of their solutions and take criticism personally on occasion. Reviews help us see how others thought process occurs and feedback given should be constructive uplifting and insightful to help others program more efficiently. Be careful to always provide feedback that does not question the person but questions how and why they created a solution the way they did. However if the developer has done something right and you found how they've written their code or how they implemented a functionality to be good/clever let them know.
 
 <p align="center">
 <img src="https://miro.medium.com/max/1400/1*D9YKlFGZKiYoR57tUTpCrg.jpeg" alt="T-Shirt Size Image" width="400">
```

Comment:
Should reference the images in the repo and modify alternative text to a more appropriate name